sys/mips/mips/pmap.c

```diff
@@ -2083,8 +2097,6 @@ pmap_enter(pmap_t pmap, vm_offset_t va, vm_page_t m, vm_prot_t prot,
 #ifdef CPU_CHERI
 	if ((flags & PMAP_ENTER_NOLOADTAGS) != 0)
 		newpte |= PTE_LC;
-	if ((flags & PMAP_ENTER_NOSTORETAGS) != 0)
-		newpte |= PTE_SC;
```

Comment:
Would it be useful to set PTE_CRO here?

Reply:
It's (now, anyway) handled via `init_pte_prot`.